mantidproject/mantid
# Error in BASISDiffraction with powder sample

**Description of work.**
- [x] `BASISCrystalDiffraction` algorithm to replace `BASISDiffraction`
- [x] `BASISDiffraction` deprecated
- [x] Created `BASISPowderDiffraction` algorithm
- [x] Documentation for `BASISPowderDiffraction` 
- [x] Documentation for `BASISCrystalDiffraction` 
- [x] System test for `BASISPowderDiffraction` 
- [x] System test for `BASISDiffraction` migrated to System test for `BASISCrystalDiffraction` 
- [x] Relase notes for `BASISPowderDiffraction`

**Report to:** Naresh Osti

**To test:**
Run the following snippet:
```
BASISPowderDiffraction(RunNumbers='74799',
                       OutputWorkspace='powder',
                       BackgroundRuns='75527',
                       OutputBackground='background'
                       VanadiumRuns='64642')
```
Verify a sharp peak for `Q~1.0`, and scattering angle `~60` degrees in workspaces `powder` and `powder_angle`, respectively.



Fixes #23380

[release notes](https://github.com/mantidproject/mantid/blob/41fa942050b30b159ec83d5d5c0391cf9e01ba74/docs/source/release/v3.14.0/indirect_inelastic.rst)

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
